src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -67,6 +67,9 @@ using grpc_core::internal::ServerRetryThrottleData;
 
 /* Client channel implementation */
 
+gpr_timespec start, end;
```

Comment:
Instead of using global variables, could record this via the disabled-by-default stats mechanism, which would give us a histogram when we run a test with a large number of RPCs.

Reply:
Do we want to keep this at all? I went the quick and dirty route thinking we'd throw this away once we had results.